compiler/scenario-service/server/src/main/scala/com/digitalasset/daml/lf/ScenarioServiceMain.scala

```diff
@@ -114,6 +116,7 @@ object ScenarioService {
     Status.NOT_FOUND.withDescription(s" context $id not found!").asRuntimeException
 }
 
+@SuppressWarnings(Array("org.wartremover.warts.NonUnitStatements"))
```

Comment:
I don't know how relevant this was in your run, but these annotations don't necessarily have to attach to a top-level definition.